src/http/verification.rs

```diff
@@ -0,0 +1,39 @@
+use html_parser::Dom;
+use std::str::FromStr;
+use url::Url;
+
+pub async fn is_verified(input_url: Url, expected_url: Url) -> bool {
+    let website = reqwest::get(input_url.to_string()).await;
+    if website.is_err() {
+        return false;
+    }
+
+    let website = website.unwrap().text().await;
+    if website.is_err() {
+        return false;
+    }
+
+    let website = Dom::parse(&website.unwrap());
+    if website.is_err() {
+        return false;
+    }
+
+    let website = website.unwrap();
+    for children in website.children {
+        if let Some(element) = children.element() {
```

Comment:
Is there a way to provide an xpath (or other path query) to the "`a` with an `href` with a `rel=me`" so that you can push the search into the (presumably faster) library and avoid having to process every element of the website linearly?